schema/omega-services.async.api.yaml

```diff
@@ -777,6 +830,199 @@ components:
       type: boolean
       description: true to indicate that this Agent is also considered an Authority File, false otherwise
 
+    RecordSummary:
+      type: object
+      required:
+        - identifier
+        - type
+        - creator
+        - current-description
+        - description
+      properties:
+        identifier:
+          $ref: '#/components/schemas/GeneralIdentifier'
+        type:
+          $ref: '#/components/schemas/RecordType'
+        creator:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralIdentifier'
```

Comment:
I think `creator` should be a `GeneralLabelledIdentifier` object as it can have a description as well as a URI.

Reply:
In the OCDM there is only a URI as far as I can see. Where are you also seeing the label come from?

Reply:
In the test data. For example:
```
cat:COAL.2022.N373.P  dct:creator [
    dct:description  "from 1965" ;
    dct:subject      cat:agent.24
] ;
```

Reply:
Erm... At a casual glance that data doesn't look correct - a Creator is an Agent (i.e. Person or Corporate Body). I can't imagine a Person or Corporate Body being described as "from 1965".